static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -492,45 +497,44 @@ const makeEpicABTest = ({
     return test;
 };
 
-const buildEpicCopy = (row: any, hasCountryName: boolean) => {
+const buildEpicCopy = (row: any): ?AcquisitionsEpicTemplateCopy => {
     const heading = throwIfEmptyString('heading', row.heading);
 
     const paragraphs: string[] = throwIfEmptyArray(
         'paragraphs',
         splitAndTrim(row.paragraphs, '\n')
     );
 
-    const countryName: ?string = hasCountryName
-        ? countryNames[geolocationGetSync()]
-        : undefined;
+    const countryName: ?string = countryNames[geolocationGetSync()];
 
-    return {
-        heading:
-            heading && countryName
+    // In case replaceCountryName throws an error
+    try {
+        return {
+            heading: heading
                 ? replaceCountryName(heading, countryName)
                 : heading,
-        paragraphs:
-            paragraphs && countryName
+            paragraphs: paragraphs
                 ? paragraphs.map<string>(para =>
                       replaceCountryName(para, countryName)
                   )
                 : paragraphs,
-        highlightedText: row.highlightedText
-            ? row.highlightedText.replace(
-                  /%%CURRENCY_SYMBOL%%/g,
-                  getLocalCurrencySymbol()
-              )
-            : undefined,
-        footer: optionalSplitAndTrim(row.footer, '\n'),
-    };
+            highlightedText: row.highlightedText
+                ? row.highlightedText.replace(
+                      /%%CURRENCY_SYMBOL%%/g,
+                      getLocalCurrencySymbol()
+                  )
+                : undefined,
+            footer: optionalSplitAndTrim(row.footer, '\n'),
+        };
+    } catch (e) {
+        return undefined;
```

Comment:
Do we want to log the error?

Reply:
I've pushed a change which logs the error in sentry in the case that the epic fails.
I've allowed the banner error to be caught further up, because the banner copy is not optional, so we cannot continue. The default banner will be displayed, in this case.

Reply:
Nice work